core/schema/module.go

```diff
@@ -99,13 +143,43 @@ func (s *moduleSchema) Install() {
 		dagql.Func("withInterface", s.moduleWithInterface).
 			Doc(`This module plus the given Interface type and associated functions`),
 
+		dagql.Func("withDependencies", s.moduleWithDependencies).
+			Doc(`Update the module configuration to use the given dependencies.`).
+			ArgDoc("dependencies", `The dependency modules to install.`),
+
+		dagql.Func("generatedSourceRootDirectory", s.moduleGeneratedSourceRootDirectory).
+			Doc(
+				`The module's root directory containing the config file for it and its source`,
+				`(possibly as a subdir). It includes any generated code or updated config files`,
+				`created after initial load, but not any files/directories that were unchanged`,
+				`after sdk codegen was run.`,
```

Comment:
Each of these args will be separated by a blank line, so this should either all be one string or be split at appropriate sentences/paragraph.

```suggestion
				`The module's root directory containing the config file for it and its source
				(possibly as a subdir). It includes any generated code or updated config files
				created after initial load, but not any files/directories that were unchanged
				after sdk codegen was run.`,
```

It'll un-word-wrap this, so one big string should be fine. Might want to do a Git-style short sentence for the first paragraph though.